src/examples/elementary/efl_thread_1.c

```diff
@@ -1,7 +1,7 @@
 //Compile with:
 //gcc -o efl_thread_1 efl_thread_1.c -g `pkg-config --cflags --libs elementary`
 #include <Elementary.h>
-#include <pthread.h>
+#include <eina_thread.h>
```

Comment:
I think that if it is not inside eina, it should include <Eina.h> and not directly <eina_thread.h>.

Reply:
oh